cfpb/consumerfinance.gov
# Use the Blog to test email signups

This change to our functional tests will use the blog’s email signup flow rather than `/consumer-tools`.

Recently we modified email signups to support either a signup link directly to GovDelivery or our existing signup box on the page. `/consumer-tools` was one of the URLs switched to a signup link, thus it is no longer useful for testing the signup flow.

The test database is updated to add an email signup to `/about-us/blog` for testing.

## How to test this PR

1. Run the email signup test spec and observe that it fails:

   ```shell
   yarn cypress run --spec test/cypress/integration/components/email-signup/email-signup.js
   ```

2. Pull this PR
3. Load the new test database:

   ```shell
   ./refresh-data.sh test.sql.gz
   ```

4. Run the email signup test spec and observe that it passes:

   ```shell
   yarn cypress run --spec test/cypress/integration/components/email-signup/email-signup.js
   ```

## Checklist
